elixir/apps/web/lib/web/components/core_components.ex

```diff
@@ -928,6 +935,28 @@ defmodule Web.CoreComponents do
     """
   end
 
+  attr :account, :any, required: true
+  attr :actor, :any, required: true
+
+  def actor_link(%{actor: %Domain.Actors.Actor{type: :api_client}} = assigns) do
+    ~H"""
+    <.link
+      class="text-accent-500 hover:underline"
```

Comment:
curious why `link_style()` isn't used here?

Reply:
It should be.  I wasn't paying attention to the class, just copy/pasted what was previously being used.  I'll update it.

Reply:
Updated